modules/ROOT/pages/proxy-latest-concept.adoc

```diff
@@ -1,30 +1,62 @@
-= Deploying an API Proxy
+= Deploy Your API Proxy
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-A proxy is an application that provides a level of protection for your API. You don't have to modify the API to guard against attacks on your web service. Mule Runtime hosts a Mule proxy application. There is a proxy type for each type of web service:
-
-* RAML/OAS Proxy
-* WSDL Proxy
-* HTTP Proxy
-* HTTPS Proxy
-
-API Manager automatically generates the proxy app when you configure the API as an endpoint with a proxy and includes an Autodiscovery element in the app. Mule locks the API until all policies have been applied. The client app (consumer) calls the proxy which forwards the call to the API. After you deploy the app, Mule Runtime calls API Manager using the client ID and secret to get the policies for the API.
-
-In most cases, the proxy you generate in API Manager is suitable for deployment. However, you can modify the proxy to log data to a file or send data to a Splunk account with the Anypoint Splunk Connector, for example.
-
-You can import a proxy that you download from API Manager into Studio as a Mule Deployable Archive. You can then modify the proxy in Studio.
-
-If you configured an endpoint with a proxy based on Mule 3 to use the HTTPS scheme, you need to modify a proxy to add HTTPS credentials following the xref:mule-runtime::build-an-https-service.adoc[Build an HTTPS Service] instructions. +
-Additionally, if you configured a proxy based on Mule 4, you can build an HTTPS proxy using a TLS context stored in Anypoint Security's secrets manager following the xref:building-https-proxy.adoc[Building an HTTPS API Proxy] instructions.
-
-
-
-Now, you're ready to deploy an auto-generated proxy.
+You can deploy your API proxy to CloudHub, Runtime Fabric, or a hybrid cloud environment. By default, the resources allocated to the proxy application include a CPU with 0.1 vCores and a memory of 500 MB. If required, modify the resource allocation from xref:runtime-manager::deploying-to-cloudhub#change-settings-of-a-deployed-application.adoc[Runtime Manager].
```

Comment:
I missed something here. This sentence: `By default, the resources allocated to the proxy application include a CPU with 0.1 vCores and a memory of 500 MB.`, applies only to Cloudhub, but we are mentioning all proxies. This value is different in Runtime Fabric, for example. And as Hybrid uses customer's infrastructure, it is constrained to that.

Reply:
Done! Thanks!